lib/Service/RoomService.php

```diff
@@ -362,6 +365,42 @@ public function setAvatar(Room $room, string $avatar): bool {
 		return true;
 	}
 
+	public function startRecording(Room $room, $status): bool {
```

Comment:
We already verified if is a moderator. Is really necessary if the moderator is on the call?

Reply:
```suggestion
	public function startRecording(Room $room, int $status): bool {
```

Reply:
>  Is it really necessary if the moderator is on the call?

I think it would be good for consistency that only people in the call can start it? But we can adjust this later.
However, checking that a call is active is a must.

But it's easy, just add:

```php
		if (!$room->getActiveSince() instanceof \DateTimeInterface) {
			throw new InvalidArgumentException('call');
		}
```